examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>Do we wanna show a Vector implementation as well?</p><p> </p><p>Are both p and on necessary? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thanks for the feedback. I'm not sure if the plan was to use a VectorFunction, but it could be implemented. @rhodrin may have some input in as well if he'd recommend I change it.